### PR TITLE
adding benefits variants

### PR DESCRIPTION
**Pull request by nshonda**
Adding a variant to pitch cards so content gets centered
related issue:
- https://airtable.com/appsEN1RgrUk3oyon/tbluJ0kJjMnu1rtsx/viwEjc1BEgM0pj72s/recQJIn1jnDLSZR3I?blocks=hide

### CODE REVIEW

**Review by quaardvark — CHANGES_REQUESTED** [review]
I think we should add a select here with a `variant` key instead of a bool. We should also treat the other "button" variant the same way, so PitchCard api would look like `<PitchCard variant={"button" | "benefits"} />`

**Review by quaardvark — CHANGES_REQUESTED** [review]
`variant` should a slice property, not each invidual pitch card

File: src/components/partials/SliceZone/components/PitchCardsSlice/index.js [hunk 1]
```diff
@@ -53,7 +55,7 @@ const PitchCardsSlice = ({ slice }) => {
       key: title?.[0]?.text || idx,
       link: linkText ? (
         <Link link={linkUrl}>
-          {linkAsButton ? (
+          {variant === "button" ? (
```
Comment (quaardvark) [inline src/components/partials/SliceZone/components/PitchCardsSlice/index.js:58]: This should also work with the old API, but maybe I missed where it's stubbed

**Comment by nshonda** [comment]
@quaardvark do you think we'll have issues with the old API if we make `variant` be a slice property since `linkAsButton` was a card property?

**Comment by quaardvark** [comment]
@nshonda All I ask is to hoist variant to top and implement this `const isButtonVariant = variant === "button" || linkAsButton`. We'll sort the API thing later, there's probably only a few pages that use it

**Comment by quaardvark** [comment]
Closing in favor of #196

**Pull request closed without merging**